lib/ace/ext/papeeria/katex-previewer.coffee

```diff
@@ -50,43 +54,142 @@ define((require, exports, module) ->
       )
       return
 
-    callbackHidePopover = ->
-      popoverHandler.hide($("#formula"))
-      editor.off("changeSelection", callbackHidePopover)
-      editor.session.off("changeScrollTop", callbackHidePopover)
-      editor.session.off("changeScrollLeft", callbackHidePopover)
-      return
+    jqEditorContainer = $(editor.container)
+    jqFormula = -> $("#formula")
+
+    # ch stands for Context Handler
+    ch = {
```

Comment:
Should I still leave the full name, considering I can't use `@` inside this object? It will make some methods really verbose. But I don't insist.